subprojects/docs/src/docs/release/notes.md

```diff
@@ -269,10 +264,13 @@ Related issues:
 
 [Support "Zulu OpenJDK Discovery API" for auto provisioning toolchains gradle#19140](https://github.com/gradle/gradle/issues/19140)
 
-#### TODO: Enhanced the plugin declaration DSL from java-gradle-plugin
+#### Enhanced the plugin declaration DSL for the java-gradle-plugin
+
+The recently released [version 1.0.0 of the Plugin-Publish plugin](https://plugins.gradle.org/plugin/com.gradle.plugin-publish/1.0.0) significantly improves the process by having stronger opinions and a more straightforward configuration. 
```

Comment:
I'm unclear what "the process" means here. Plugin declaration? Plugin publishing? Both?